src/main/java/ch/bfh/bti7081/s2018/green/presenters/MedicationPrescriptionPresenter.java

```diff
@@ -0,0 +1,129 @@
+package ch.bfh.bti7081.s2018.green.presenters;
+
+import ch.bfh.bti7081.s2018.green.models.entities.Medication;
+import ch.bfh.bti7081.s2018.green.views.MedciationPrescriptionView;
+import ch.bfh.bti7081.s2018.green.views.enumerations.TxtBtnSaveEdit;
+import com.vaadin.ui.*;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+public class MedicationPrescriptionPresenter extends MedciationPrescriptionView {
+
+    public MedicationPrescriptionPresenter (Medication med, CustomLayout body) {
+        super(med, body);
+        if (med == null) addCreateNewButtonListener(TxtBtnSaveEdit.SAVE.toString());
```

Comment:
Please always use curly braces and line breaks

Reply:
code.ifElse.optional.hasCurlyBracesStlye() ? system.out.println("I dont like too many lines of code  - what about you?") : system.out.println("Why do you don't like this?");